aws_lambda_powertools/event_handler/appsync.py

```diff
@@ -1,21 +1,32 @@
 import logging
-from typing import Any, Callable, Optional, Type, TypeVar
+from itertools import groupby
+from typing import Any, Callable, List, Optional, Type, Union
 
 from aws_lambda_powertools.utilities.data_classes import AppSyncResolverEvent
 from aws_lambda_powertools.utilities.typing import LambdaContext
 
 logger = logging.getLogger(__name__)
 
-AppSyncResolverEventT = TypeVar("AppSyncResolverEventT", bound=AppSyncResolverEvent)
 
+class RouterContext:
```

Comment:
Nice!! Worth checking whether a `Router` instance still gets the Context injected correctly (in addition to AppSyncResolver) - our functional tests should cover it already.

Reply:
Correct, functional tests have it covered and it works as expected